WalletWasabi.Backend/Startup.cs

```diff
@@ -50,14 +58,15 @@ public void ConfigureServices(IServiceCollection services)
 			services.AddLogging(Logging => Logging.AddFilter((s, level) => level >= Microsoft.Extensions.Logging.LogLevel.Warning));
 
 			services.AddSingleton<IExchangeRateProvider>(new ExchangeRateProvider());
-
+			services.AddSingleton<Global>(new Global(Configuration["datadir"]));
```

Comment:
Why do we need getting the `datadir` from a different config file? Is it for testing?

Reply:
yes, so `Regtests` does not use the same config file than the user's datadir.